examples/kubernetes/spark/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.5-slim-buster
```

Comment:
Could you use newer python images like 3.6 and 3.7 as we dropped the support for python 3.5, please?